splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/util/messageUtil.js

```diff
@@ -0,0 +1,18 @@
+import * as __ from 'lodash';
+import { _ } from '@splunk/ui-utils/i18n';
+import messageDict from '../constants/messageDict';
+
+/**
+ * @param code {Int} a int value.
```

Comment:
We should follow this docstring pattern everywhere.

Reply:
Okay

Reply:
In this PR most of the javascript code is taken from the swc. So next time will add the docstring pattern in the function if necessary